File: sportsbook-app/wallet-service/src/main/java/com/sportsbook/walletservice/controller/SampleController.java
package com.sportsbook.walletservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SampleController {

    @GetMapping("/api/walletservice/ping")
    public String ping() {
        return "wallet-service pong";
    }
}

File: sportsbook-app/wallet-service/src/main/kotlin/com/sportsbook/walletservice/controller/SampleController.kt
package com.sportsbook.walletservice.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class SampleController {

    @GetMapping("/api/walletservice/ping")
    fun ping(): String = "wallet-service pong"
}
